### PR TITLE
Changed to able to get/update 'aliases' parameter of HOST record for DNS

### PR DESCRIPTION
**Pull request by userlocalhost**
Thank you make this useful library! Please let me enhance a feature for DNS.

## Background
Current implementation couldn't touch with `aliases` field of HOST record for DNS.

## Change 
This patch enables to get/set it.

### CODE REVIEW

**Comment by saiprasannasastry** [comment]
LGTM

File: HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 History
 -------
 
+0.4.22 (2019-02-05)
+___________________
+
+* Changed to able to get/update 'aliases' parameter of HOST record for DNS
```
Comment (saiprasannasastry) [inline HISTORY.rst]: can this changes be removed, because we are planning to merge another pr and bump the version.

Comment (userlocalhost) [reply HISTORY.rst]: Thank you for your review.

> we are planning to merge another pr and bump the version.

I got it. After merging it, I'll merge latest master branch and resolve conflicts of these files (`HISTORY.rst` and `infoblox_client/__init__.py`) properly.
Please go ahead.

Comment (userlocalhost) [reply HISTORY.rst]: Thank you for your review.

> we are planning to merge another pr and bump the version.
I got it. After merging it, I'll merge latest master branch and resolve conflicts of these files (`HISTORY.rst` and `infoblox_client/__init__.py`) properly.
Please go ahead.

Comment (saiprasannasastry) [reply HISTORY.rst]: no i meant remove the History.rst from this pr, i will update it when we are planning to bump the version

File: infoblox_client/__init__.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 __author__ = 'John Belamaric'
 __email__ = 'jbelamaric@infoblox.com'
-__version__ = '0.4.21'
+__version__ = '0.4.22'
```
Comment (saiprasannasastry) [inline infoblox_client/__init__.py]: you have missed it here

Comment (saiprasannasastry) [reply infoblox_client/__init__.py]: it should be 0.4.21

Comment (userlocalhost) [reply infoblox_client/__init__.py]: Sorry, I'll remove last two commits (back to a24ad1a)

**Pull request merged**